core/src/com/unciv/logic/automation/ConstructionAutomation.kt

```diff
@@ -187,6 +186,11 @@ class ConstructionAutomation(val cityConstructions: CityConstructions){
         if (preferredVictoryType == VictoryType.Cultural
                 && wonder.name in listOf("Sistine Chapel", "Eiffel Tower", "Cristo Redentor", "Neuschwanstein", "Sydney Opera House"))
             return 3f
+        // Only start building if we are the city that would complete it the soonest
+        if (wonder.name == "Utopia Project" && cityInfo == civInfo.cities.minByOrNull { 
+                it.cityConstructions.getRemainingWork(it.cityConstructions.getCurrentConstruction().name) / cityInfo.cityStats.currentCityStats.production.roundToInt() + wonder.cost / it.cityStats.currentCityStats.production 
```

Comment:
I don't understand the calculation here, something seems confused

Reply:
What is does is basically:

For each city:
    Calculate the amount of turns remaining for building the current construction
    Add to this the amount of turns required to build the Utopia Project
If the current city has the lowest amount of turns among all cities:
    Start building the utopia project here

Reply:
Looking at it further I did use the wrong variable once, should be fixed in the next commit